ReactAndroid/build.gradle

```diff
@@ -289,6 +299,10 @@ android {
 
         buildConfigField("boolean", "IS_INTERNAL_BUILD", "false")
         buildConfigField("int", "EXOPACKAGE_FLAGS", "0")
+
+        resValue "integer", "react_native_dev_server_port", reactNativeDevServerPort()
+        resValue "integer", "react_native_inspector_proxy_port", reactNativeInspectorProxyPort()
```

Comment:
Updated. This now lets you set the inspector port using:

```
./gradlew :app:installDebug -PreactNativeDevServerPort=3000
```

By default it uses the dev server port.